Task5.py

```diff
@@ -0,0 +1,22 @@
+"""RegularExpressions."""
+
+
+import re
+
+text = input("Enter text: " + '\n')
+expression = re.compile(r"(\S+@\S+)")
```

Comment:
This pattern will count `adf@asdf` as valid email but it isn't( it should ended with .domain)